### ISSUE TITLE
Markdown Support

### ISSUE THREAD

**Issue by KannaDev** [HDInnovations/UNIT3D#407]
### Issue Type
- [ ] Bug Report
- [ ] Feature Idea
- [x] Enhancement
- [ ] Technical Discussion
- [ ] Question

Requesting markdown support for chat and uploads. Markdown is widely used and is more convenient to use in my opinion than BBCODE.

**Comment by HDVinnie** [comment]
I agree with you....markdown is more popular in general. However when it comes to tracker platforms they all use BBCODE which is why I went with it. However it could be possible to add markdown support on top of the existing BBCODE support.

**Comment by KannaDev** [comment]
that would be great xD
